diff -urN a/conduit-packages-2.lisp b/conduit-packages-2.lisp
--- a/conduit-packages-2.lisp	1969-12-31 16:00:00.000000000 -0800
+++ b/conduit-packages-2.lisp	2016-11-11 15:30:01.988535295 -0800
@@ -0,0 +1,37 @@
+(in-package :org.tfeb.conduit-packages)
+;;;; Define the CL/CONDUITS package and a user package.
+;;;
+;;; I would like to be able to say simply (EVAL-WHEN (:LOAD-TOPLEVEL
+;;; ...)  ...)  here, but that breaks, because that results in
+;;; DEFPACKAGE being processed as a top-level form in not-compile-time
+;;; mode, and *it* expands to (EVAL-WHEN (:COMPILE-TOPLEVEL ...) ...),
+;;; so actually gets evaluated at compile-time, which fails.  (LET ()
+;;; ...) is just enough to stop this: LOCALLY or PROGN is not.  This is
+;;; broken.
+;;;
+;;; CLISP 2000-03-06 (March 2000)  can't hack this at all:  you need to
+;;; extract the remainder of this file and put it into a different
+;;; file, compiled and loaded after the main file is loaded.
+;;;
+;;; CMUCL 18b can't do this.  CMUCL 18c Sources 2000-09-27 does have
+;;; bugs in EVAL-WHEN, but does this right.
+;;;
+(defpackage :org.tfeb.cl/conduits
+  (:use)
+  (:nicknames :org.tfeb.clc)
+  (:extends/excluding :cl #:export #:unexport #:defpackage 
+                      #:delete-package #:rename-package)
+  (:extends/excluding :org.tfeb.conduit-packages
+                      #:recompute-conduits))
+(defpackage :org.tfeb.cl-user/conduits
+      (:nicknames :org.tfeb.clc-user)
+      (:use :org.tfeb.clc))
+
+#+Genera
+(pushnew (find-package :org.tfeb.cl/conduits) si:*reasonable-packages*)
+
+#||
+(defpackage :cl/magic-if
+  (:extends/excluding :cl #:if)
+  (:export #:if))
+||#
diff -urN a/conduit-packages.asd b/conduit-packages.asd
--- a/conduit-packages.asd	2016-11-11 15:07:59.384153628 -0800
+++ b/conduit-packages.asd	2016-11-11 15:26:44.883847371 -0800
@@ -1,5 +1,7 @@
 ;;;; conduit-packages.asd, automatically created by quicklisp
 
 (asdf:defsystem #:conduit-packages
+  :serial t
   :components
-  ((:file "conduit-packages")))
+  ((:file "conduit-packages")
+   (:file "conduit-packages-2")))
diff -urN a/conduit-packages.lisp b/conduit-packages.lisp
--- a/conduit-packages.lisp	2016-11-11 15:07:59.384153628 -0800
+++ b/conduit-packages.lisp	2016-11-11 15:10:29.458378369 -0800
@@ -415,41 +415,3 @@
 	(nsubstitute new-name name c))
       (dolist (p *package-conduits*)
       	(nsubstitute new-name name p)))))
-
-;;;; Define the CL/CONDUITS package and a user package.
-;;;
-;;; I would like to be able to say simply (EVAL-WHEN (:LOAD-TOPLEVEL
-;;; ...)  ...)  here, but that breaks, because that results in
-;;; DEFPACKAGE being processed as a top-level form in not-compile-time
-;;; mode, and *it* expands to (EVAL-WHEN (:COMPILE-TOPLEVEL ...) ...),
-;;; so actually gets evaluated at compile-time, which fails.  (LET ()
-;;; ...) is just enough to stop this: LOCALLY or PROGN is not.  This is
-;;; broken.
-;;;
-;;; CLISP 2000-03-06 (March 2000)  can't hack this at all:  you need to
-;;; extract the remainder of this file and put it into a different
-;;; file, compiled and loaded after the main file is loaded.
-;;;
-;;; CMUCL 18b can't do this.  CMUCL 18c Sources 2000-09-27 does have
-;;; bugs in EVAL-WHEN, but does this right.
-;;;
-(eval-when (:load-toplevel :execute)
-  (let ()
-    (defpackage :org.tfeb.cl/conduits
-      (:use)
-      (:nicknames :org.tfeb.clc)
-      (:extends/excluding :cl #:export #:unexport #:defpackage 
-			  #:delete-package #:rename-package)
-      (:extends/excluding :org.tfeb.conduit-packages
-       #:recompute-conduits))
-    (defpackage :org.tfeb.cl-user/conduits
-      (:nicknames :org.tfeb.clc-user)
-      (:use :org.tfeb.clc)))
-  #+Genera
-  (pushnew (find-package :org.tfeb.cl/conduits) si:*reasonable-packages*))
-
-#||
-(defpackage :cl/magic-if
-  (:extends/excluding :cl #:if)
-  (:export #:if))
-||#
