diff -urN a/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.asd b/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.asd
--- a/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.asd	2016-11-11 14:33:58.701238910 -0800
+++ b/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.asd	2016-11-11 14:40:53.552350520 -0800
@@ -33,6 +33,8 @@
 ;;;;**************************************************************************
 
 
+#+clisp(eval-when (:compile-toplevel :load-toplevel :execute) (setf custom:*ansi* t))
+
 (eval-when (:compile-toplevel :load-toplevel :execute)
   #+clisp
   (unless custom:*ansi*
diff -urN a/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.test.asd b/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.test.asd
--- a/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.test.asd	2016-11-11 14:33:58.701238910 -0800
+++ b/common-lisp/cesarum/com.informatimago.common-lisp.cesarum.test.asd	2016-11-11 14:41:05.696207330 -0800
@@ -32,6 +32,8 @@
 ;;;;    along with this program.  If not, see http://www.gnu.org/licenses/
 ;;;;**************************************************************************
 
+#+clisp(eval-when (:compile-toplevel :load-toplevel :execute) (setf custom:*ansi* t))
+
 (eval-when (:compile-toplevel :load-toplevel :execute)
   #+clisp
   (unless custom:*ansi*
